rllib/models/tests/test_base_model.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+
+from ray.rllib.models.base_model import (
+    SpecDict,
+    UnrollOutputType,
+    TensorDict,
+    Model,
+    RecurrentModel,
+    ForwardOutputType,
+)
+
+
+class NoopRecurrentModelImpl(RecurrentModel):
+    input_spec = output_spec = next_state_spec = prev_state_spec = SpecDict()
+
+    def _initial_state(self):
+        return TensorDict()
+
+    def _unroll(self, inputs: TensorDict, prev_state: TensorDict) -> UnrollOutputType:
+        return (TensorDict(), TensorDict())
```

Comment:
Can you actually fill these with some numpy values? choose numpy over torch to emphasize framework agnostic-ness.

Reply:
feed in the initial state to the constructor and compare one source of truth in the actual test.

```
# something like this.
rnn = NoopRecurrentModelImpl(init_state=np.array([1, 2]))
assert rnn.initial_state() == np.array([1, 2])
```